### PR TITLE
Merge upstream train

### PR DESCRIPTION
**Pull request by brtknr**

### CODE REVIEW

File: ansible/roles/prometheus/defaults/main.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ prometheus_services:
     dimensions: "{{ prometheus_server_dimensions }}"
     haproxy:
       prometheus_server:
-        enabled: "{{ enable_prometheus_server }}"
+        enabled: "{{ enable_prometheus | bool }}"
```
Comment (markgoddard) [inline ansible/roles/prometheus/defaults/main.yml:14]: Should be enable_prometheus_server

**Review by priteau — DISMISSED** [review]
Looks OK to me, although it's missing a few of the most recent commits.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: ansible/roles/prometheus/defaults/main.yml (hunk 1 above)
Comment (brtknr) [reply ansible/roles/prometheus/defaults/main.yml:14]: This is what came from upstream though

Comment (brtknr) [reply ansible/roles/prometheus/defaults/main.yml:14]: From commit 55b2eee95b54880ee0656bef1f771eddee7cd7d9

Comment (priteau) [reply ansible/roles/prometheus/defaults/main.yml:14]: The stackhpc branch has a backport of the prometheus server disable commit: https://github.com/stackhpc/kolla-ansible/commit/078b6f749c6b5092f18bd7bf35db79377694f036

Comment (brtknr) [reply ansible/roles/prometheus/defaults/main.yml]: The latest tip of this branch has all the missing commits and uses enable_prometheus_server

**Review by markgoddard — APPROVED** [review]

**Comment by brtknr** [comment]
Thank you!

**Pull request merged**